odbc/src/api/functions.rs

```diff
@@ -2237,6 +2211,12 @@ unsafe fn sql_get_infow_helper(
                         string_length_ptr,
                     )
                 }
+                InfoType::SQL_MAX_CONCURRENT_ACTIVITIES => {
```

Comment:
Umh, this seems to imply that we only support 1 statement at a time.
Is this true? I thought we could have multiple statements, since we don't rely on static/shared things inside statements (all is in the statement struct I think)?
It might be why you see PowerBI opening a bunch of new connections for each statement.

Reply:
I've set this to 10, which is our configured connection pool size for the driver.